modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -56,11 +56,13 @@ spec:
 [crio.runtime.workloads.management]
 activation_annotation = "target.workload.openshift.io/management"
 annotation_prefix = "resources.workload.openshift.io"
-resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" } <1>
+[crio.runtime.workloads.management.resources]
+cpushares = 0
+CPUs = "0-1, 52-53" <1>
 ----
-<1> The `cpuset` value will vary based on the installation.
+<1> The `CPUs` value will vary based on the installation.
```

Comment:
I see this is existing tet, but we try to avoid _will_ (for some reason).

s/will vary/varies